### PR TITLE
closes #13, Mapping links to name

### LINKED ISSUE #13: prettify the domain links in the sidebar
The domains in the side bar should be displayed in a prettier format as indicated below:

https://bitcointalk.org == bitcointalk

https://bitcoin.stackexchange.com == Bitcoin StackExchange

https://lists.linuxfoundation.org/pipermail/bitcoin-dev/ == bitcoin dev mailing list

https://lists.linuxfoundation.org/pipermail/lightning-dev/ == lightning dev mailing list

https://btctranscripts.com/ == BTC transcripts

https://bitcoinops.org/en/ == Bitcoin Optech

### PR DESCRIPTION
**Pull request by Emmanuel-Develops**
I cloned elastic search MultiCheckbox facet and modified the getFilterValueDisplay Function. mapping.json maps the links to their respective custom names. 

A better approach would be if both name and value came from the backend. Until then, this remains a viable option.

### CODE REVIEW

File: src/components/CustomMultiCheckboxFacet.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          aria-label="Show more options"
+        >
+          + More
+        </button>
+      )}
+    </fieldset>
+  );
+}
+
+export default CustomMultiCheckboxFacet;
```
Comment (adamjonas) [inline src/components/CustomMultiCheckboxFacet.js:112]: EOF empty line needed here

File: src/config/mapping.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "labels": {
+        "https://bitcointalk.org": "bitcointalk",
+        "https://bitcoin.stackexchange.com": "Bitcoin StackExchange",
+        "https://lists.linuxfoundation.org/pipermail/bitcoin-dev/": "Bitcoin dev mailing list",
+        "https://lists.linuxfoundation.org/pipermail/lightning-dev/": "Lightning dev mailing list",
+        "https://btctranscripts.com/": "BTC transcripts",
+        "https://bitcoinops.org/en/": "Bitcoin Optech"
+    }
+}
```
Comment (adamjonas) [inline src/config/mapping.json:10]: EOF empty line

**Comment by adamjonas** [comment]
Is that possible to do from the backend?

**Review by adamjonas — COMMENTED** [review]
Given your modification to ES MultiCheckbox can you comment where you are modifying it so we know what is custom and what is standard?

Good for merge after that and the EOF empty line additions. Thanks!

**Comment by Emmanuel-Develops** [comment]
> 

I have included the modified function in a comment, it is a view component so it doesn't have much logic. I cloned it because it's faster that way than writing a new view component while keeping the same styling. Given that it's been extrapolated to a custom component it will be much easier to modify it going forward.

**Comment by Emmanuel-Develops** [comment]
> > 
> 
> I have included the modified function in a comment, it is a view component so it doesn't have much logic. I cloned it because it's faster that way than writing a new view component while keeping the same styling. Given that it's been extrapolated to a custom component it will be much easier to modify it going forward.

Whether backend can provide that depends on elastic search, looking at the elastic search response structure the type "value" (what we use for domain) doesn't have a name field. I'll do more research to determine if it can be included

**Pull request merged**